docs/pages/tutorial.md

```diff
@@ -594,8 +594,58 @@ import requests, json;
 "
 ```
 
+## App Template
+
+The best way to start building a new application is to use [Aana App Template](https://github.com/mobiusml/aana_app_template). It is a GitHub template repository that you can use to create a new repository with the same directory structure and files as the template. It will help you get started with the Aana SDK and provide you with a basic structure for your application and its dependencies.
+
+Let's create a new project using the Aana App Template:
+
+### Create a New Repository
+
+Go to the [Aana App Template](https://github.com/new?template_name=aana_app_template). Choose a name for your repository and fill in the other details. Click on the "Create repository" button. GitHub will create a new repository with the same directory structure and files as the template. There is a GitHub Actions workflow that will run after the repository is created to modify the project name to match the repository name. It is pretty fast but make sure it's finished before you proceed.
+
+### Clone the Repository
+
+Clone the repository to your local machine:
+
+```bash
+git clone <repository-url>
+cd <repository-name>
+```
+
+### Create the Application
+
+For our example project we need to adjust a few things.
+
+#### Deployments
+
+Define the deployments in the [`aana_summarize_video/configs/deployments.py`](https://github.com/mobiusml/aana_summarize_video/blob/main/aana_summarize_video/configs/deployments.py). Just like in the example above.
```

Comment:
It's better to mention the file configs/deployments.py then pointed to the example

Reply:
I'm not sure I understand. Can you clarify what you mean? The example is made for this tutorial, it's just the code from the tutorial as an Aana App project. So pointing to the deployments.py I want to show what that transition (code from the tutorial -> app project) looks like. Keep in mind that `configs/deployments.py` here is not the same as `configs/deployments.py` in the Aana SDK. `configs/deployments.py` in the Aana SDK is about to be deleted.

Reply:
something like this:
`define your deployments in [configs/deployments.py](https://github.com/mobiusml/aana_app_template/blob/main/aana_app_project/configs/deployments.py) file like this example [`aana_summarize_video/configs/deployments.py`](https://github.com/mobiusml/aana_summarize_video/blob/main/aana_summarize_video/configs/deployments.py)

Reply:
Done. Can you take a look now?